cllc-public-app/go-cllc.sh

```diff
@@ -0,0 +1,22 @@
+# setup environment and then run dotnet with whatever passed params\
```

Comment:
This file was accidentally commited.  Can you ammend your commit and remove it; it has application secrets in it.